moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -732,7 +732,7 @@ bool MoveItConfigData::outputControllersYAML(const std::string& file_path)
   std::ofstream output_stream(file_path.c_str(), std::ios_base::trunc);
   if (!output_stream.good())
   {
-    ROS_ERROR_STREAM("Unable to open file for writing " << file_path);
+    ROS_ERROR_STREAM_NAMED("ros_controllers_config", "Unable to open file for writing " << file_path);
```

Comment:
how about namespace = "setup_assistant"

Reply:
-1 The package name (which is moveit_setup_assistant) is already part of the logger name. No need for an additional name at all!

Reply:
Good point about the duplicate namespace, I should have recommended the file name "moveit_config_data" as that's the standard we usually follow. I really do like using _NAMED though. I've just attempted to document this standard better:
https://github.com/ros-planning/moveit.ros.org/pull/166

Reply:
So do we get back to `ROS_ERROR_STREAM`?

Reply:
That's my suggestion. @davetcoleman?

Reply:
I prefer the consistency of always using NAMED as I've just attempted to document in https://github.com/ros-planning/moveit.ros.org/pull/166. Perhaps a Github issue should be opened to specifically debate this. Either way, having the namespace certainly does not hurt anything so I would like to just merge this PR.

Reply:
@davetcoleman I don't agree. If you just have a redundant logger name like xxx.xxx, there is no reason to insist in `NAMED`. When we recently reworked the logging in some other MoveIt packages, we also dropped the extra NAME in such cases.